node/core/pvf/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+name = "polkadot-node-core-pvf"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[[bin]]
+name = "puppet_worker"
+path = "bin/puppet_worker.rs"
+
+[dependencies]
+always-assert = "0.1"
+async-std = { version = "1.8.0", features = ["attributes"] }
```

Comment:
Is it OK to mix `async-std` and tokio?

Reply:
uh-oh!! 

I assumed the effective executor (i.e. the stuff that was spawned with ctx.spawn_blocking) is async-std, but I never actually checked. So you are saying it is tokio?

Reply:
I don't know for sure 🙈

Reply:
Ok, this shouldn't be a problem. Apparently `async_std` should work on all executors.